src/libraries/System.Private.CoreLib/src/System/Threading/Tasks/ThreadPoolTaskScheduler.cs

```diff
@@ -45,17 +45,12 @@ protected internal override void QueueTask(Task task)
             if (Thread.IsThreadStartSupported && (options & TaskCreationOptions.LongRunning) != 0)
             {
                 // Run LongRunning tasks on their own dedicated thread.
-                Thread thread = new Thread(s_longRunningThreadWork);
-                thread.IsBackground = true; // Keep this thread from blocking process shutdown
-#if !TARGET_BROWSER
-                thread.Start(task);
-#endif
+                new Thread(s_longRunningThreadWork) { IsBackground = true }.UnsafeStart(task);
```

Comment:
This might warn on for mono browser build, that is why we had the if def, maybe suppress the warning?

Reply:
Ah ok. It may be better to disable the warning instead of the ifdef to make it self-documenting.

Reply:
The ifdef as it was written is broken.  It would mean that in the browser code that did:
```C#
Task.Factory.StartNew(() => DoStuff(), TaskCreationOptions.LongRunning);
```
would nop rather than eventually running DoStuff.

I will suppress the warning.  Thanks.